### PR TITLE
cli: make sure tilt down fails properly if the kubectl/docker-compose commands fail

### PR DESCRIPTION
**Pull request by nicks**
Hello @maiamcc,

Please review the following commits I made in branch nicks/ch2853:

ff12f2afc3121f203b042401f1f6dfc76067b071 (2019-07-05 13:40:06 -0400)
cli: make sure tilt down fails properly if the kubectl/docker-compose commands fail

### CODE REVIEW

File: internal/cli/down.go [hunk 1]
```diff
@@ -21,7 +20,8 @@ type downCmd struct {
 func (c *downCmd) register() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "down",
-		Short: "delete kubernetes resources",
+		Short: "delete resources created by 'tilt up'",
+		Args:  cobra.NoArgs,
```
Comment (maiamcc) [inline internal/cli/down.go]: hang on, this should be able to take args -- e.g. if someone is using the `--file` arg to specify a non-standard Tiltfile path, or if you only `tilt up`'d services A and B, and so only want to `tilt down` those specific ones, lest you get an error.

File: internal/cli/down.go [hunk 2]
```diff
@@ -64,10 +66,14 @@ func (c *downCmd) run(ctx context.Context, args []string) error {
 			break
 		}
 	}
-	dcc := downDeps.dcClient
-	err = dcc.Down(ctx, dcConfigPaths, logger.Get(ctx).Writer(logger.InfoLvl), logger.Get(ctx).Writer(logger.InfoLvl))
-	if err != nil {
-		logger.Get(ctx).Infof("error running `docker-compose down`: %v", err)
+
+	if len(dcConfigPaths) > 0 {
```
Comment (maiamcc) [inline internal/cli/down.go]: ha i was gonna leave this for maria to fix but okay!

Comment (nicks) [reply internal/cli/down.go]: i couldn't fix the bug without also fixing this problem

Comment (nicks) [reply internal/cli/down.go]: (or the integration tests would have failed)

File: internal/cli/down.go (hunk 1 above)
Comment (nicks) [reply internal/cli/down.go]: re: the --file arg

this still works fine. cobra.NoArgs only applies to positional args

https://godoc.org/github.com/spf13/cobra#PositionalArgs

re: if you only tilt up'd services A and B, and so only want to tilt down those specific ones

that seems reasonable, but that doesn't work in the current code. do you think we should file a feature request for it now, or when someone requests it?

File: internal/cli/down.go (hunk 2 above)
Comment (maiamcc) [reply internal/cli/down.go]: i figured as much :)

File: internal/cli/down.go (hunk 1 above)
Comment (maiamcc) [reply internal/cli/down.go]: ah, gotcha. yeah let's wait until someone complains!

**Pull request merged**